python/tvm/relax/frontend/onnx/onnx_frontend.py

```diff
@@ -100,14 +100,17 @@ def get_constant(
         return var
 
 
-def get_info(info_proto: onnx.onnx_ml_pb2.ValueInfoProto) -> Tuple[str, List, str, List]:
+def get_info(info_proto: onnx.onnx_ml_pb2.ValueInfoProto, value_dict={}) -> Tuple[str, List, str, List]:
```

Comment:
if `value_dict` is the output of the function, please move it to the return values, instead of reference param

Reply:
`value_dict` should be the input of the function. When a symbol appears for the second time, the value in the dict should be used.